tests/lib/hardware/libvirt.py

```diff
@@ -365,21 +365,26 @@ def _create_network(self):
             dhcp_start = str(netaddr.IPAddress(network.first+2))
             dhcp_end = str(netaddr.IPAddress(network.last-1))
             xml = textwrap.dedent("""
-                <network>
-                <name>%(network_name)s</name>
-                <forward mode="nat"/>
-                <ip address="%(host_ip)s" netmask="%(netmask)s">
-                    <dhcp>
-                        <range start="%(dhcp_start)s" end="%(dhcp_end)s" />
-                    </dhcp>
-                </ip>
-                </network>
-            """ % {
+    <network xmlns:dnsmasq='http://libvirt.org/schemas/network/dnsmasq/1.0'>
```

Comment:
Actually I changed the current indent because it would not allow longer line.
I find it more readable to have most of the line on a single one.

Reply:
Instead of playing with dedenting and indentation, could you please move the template to a separate file and load if required. Use `__file__` to determine relative path to it.

Reply:
Up until now it was preferred to have template within the code so I'm not sure about it, even though I tend to prefer it as you describe.

Reply:
Not sure, I never prefer mulitline xml code having in python module.

Reply:
If I saw the initial commit on review, I asked to move it out.